File: A1/A1-034/A1-034.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int n;
    cin >> n;
    // Find minimum
    int num, minimum;
    cin >> num;
    minimum = num;
    for(int i = 1; i < n; i++)
    {
        cin >> num;
        minimum = min(minimum, num);
    }
    // Output
    cout << minimum;
    return 0;
}

File: A1/A1-034/A1-034.py
# --------------------------------------------------
# File Name : A1-034.py
# Problem   : Minimum Value
# Author    : Worralop Srichainont
# Date      : 2026-03-13
# --------------------------------------------------

# Input amount of numbers
n_numbers = int(input().strip())

# Find the minimum numbers
min_number = int(input().strip())
for _ in range(n_numbers - 1):
    current_number = int(input().strip())
    min_number = min(min_number, current_number)

# Output the minimum number
print(min_number)
